asabiris/exceptions.py

```diff
@@ -1,2 +1,37 @@
 class SMTPDeliverError(Exception):
 	pass
+
+
+class PathError(Exception):
+	"""
+
+	Equivalent to HTTP 404 Not-Found.
+	"""
+
+	def __init__(self, message=None, *args, path=None):
+		self.Path = path
+		if message is not None:
+			super().__init__(message, *args)
+		elif path is not None:
+			message = "Invalid path {!r}.".format(path)
+			super().__init__(message, *args)
+		else:
+			super().__init__(message, *args)
+
+
+class FormatError(Exception):
```

Comment:
@ateska I have a new exception when there is a presence of format html/pdf while dealing with templates.

```
{
    "to": ["foo@example.com"],
     "body":{
      "template":"/Templates/Emaifl/hello.md",
      "params":{
         "name":"Iris"
      }
   },
    "attachments":[
        {
            "template":"/Templates/Emfail/hello.md",
            "params":{
                "name":"Iris"
            },       
            "format":"pdf"
        }
      ]
   }

```

Reply:
ok